### PR TITLE
Swat tool update

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR) updates the first draft of the Site-Wide Analysis Tool topic with additional instructions and screenshots.

## Magento release version

- [x] 2.4.1

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only
- [x] Magento Commerce Cloud

Is this update specific to an installed feature extension

- [ ] B2B extensionhttps://docs.magento.com/user-guide/reports/customer-engagement.html
- [ ] Other feature set, please specify:

## Affected documentation pages

- [Site Wide Analysis Tool](https://docs.magento.com/user-guide/reports/site-wide-analysis-tool.html) (new)

## Additional information

Staging: #4

### CODE REVIEW

File: src/reports/site-wide-analysis-tool.md [hunk 1]
```diff
@@ -2,12 +2,53 @@
...
+
+1. For Admin users with custom roles, assign permission to access the _System Insights_ [Site-Wide Analysis Tool]({% link system/permissions-user-roles.md %}) role resource.
+
+1. If necessary, notify any users who are assigned the custom role to sign out of the Admin, and sign in again.
+
+## Step 2: Access SWAT
+
+1. On the Admin sidebar, go to **Reports** > System Insights > **Site-Wide-Analysis Tool**.
+
+1. Read the _Terms of Use_ for the _Site Wide Analysis Tool_.  To continue, click <span class="btn">Accept</span>.
```
Comment (leslietilling) [inline src/reports/site-wide-analysis-tool.md:24]:
```suggestion
1. Read the _Terms of Use_ for the Site Wide Analysis Tool and click <span class="btn">Accept</span> to continue.
```

File: src/reports/site-wide-analysis-tool.md [hunk 2]
```diff
@@ -2,12 +2,53 @@
...
+1. Select the checkbox for each **Type** and **Priority** setting that you want to include in the report.
+
+1. Click <span class="btn">Generate Report</span>.
+
+   ![Report Settings]({% link images/images-ee/reports-swat-report-settings.png %}){: .zoom}
+   _Report Settings_
+
+|Tab|Description|
+|--- |--- |
+|Dashboard|Shows the health of your system with current notifications and recommendations by priority|
```
Comment (leslietilling) [inline src/reports/site-wide-analysis-tool.md:49]:
```suggestion
|Dashboard|Shows the health of your system with current notifications and recommendations by priority.|
```

**Review by leslietilling — APPROVED** [review]
A couple of minor tweaks then good.

**Comment by ccondra** [comment]
running tests

**Pull request merged**